### PR TITLE
Added new HLD files and related images 

### PR DESCRIPTION
**Pull request by TechWriter-Mayur**
Added new HLD files. They are:
doc/HLD_of_FDMI.rst
doc/HLD_of_Function_Shipping_and_In-Storage_Compute.rst
doc/HLD_of_Spiel_API.rst

Signed-off-by: TechWriter-Mayur <mayur.gupta@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: doc/HLD_of_Auxiliary_Databases_for_SNS_repair.rst [hunk 1]
```diff
@@ -0,0 +1,300 @@
+High level design of Auxiliary Databases for SNS repair
```
Comment (huanghua78) [inline doc/HLD_of_Auxiliary_Databases_for_SNS_repair.rst:1]: This document is already available at 
https://github.com/Seagate/cortx-motr/blob/main/doc/HLD_of_Auxillary_Databases.rst

Is this the same or similar document?

Comment (TechWriter-Mayur) [reply doc/HLD_of_Auxiliary_Databases_for_SNS_repair.rst:1]: `

> This document is already available at
> https://github.com/Seagate/cortx-motr/blob/main/doc/HLD_of_Auxillary_Databases.rst
> 
> Is this the same or similar document?

Yes, this is the same document. The information is not updated in the following document: https://github.com/Seagate/cortx-motr/blob/main/doc/motr-design-doc-list.rst 

So, I got confused and created the file. But the available file requires few additional changes I will update it. Meanwhile let me remove this file from the PR.

**Review by huanghua78 — APPROVED** [review]
Looks fine.

**Review by Saumya-Sunder — APPROVED** [review]

**Pull request merged**